docs/book/releases.md

```diff
@@ -0,0 +1,57 @@
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD004 -->
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver Releases
+
+- [v2.1.0](#v2.1.0)
+- [v2.0.1](#v2.0.1)
+- [v2.0.0](#v2.0.0)
+- [v1.0.2](#v1.0.2)
+- [v1.0.1](#v1.0.1)
+
+## v2.1.0<a id="v2.1.0"></a>
+
+* Refer https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.1.0 for features listing and notable changes.
+* Known issues with v2.1.0 release
+  * `v2.1.0` has issue with static file volume provisioning. When the file PV is re-created with the same PV name, volume is not getting registered as a container volume with vSphere and thus further operations like attach/delete can not be performed on the volume.
+  * `v2.1.0` has a bug in the metadata-syncer container which can delete the volume physically upon PV deletion even when PV is marked with reclaim policy `retain`.
+* Deployment YAML files - https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/release-2.1/manifests/v2.1.0
+* Requires minimum Kubernetes version `v1.17.0`
+
+## v2.0.1<a id="v2.0.1"></a>
+
+* Refer https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.0.1 for feature listing and notable changes.
+* Known issues with v2.0.1 release
```

Comment:
We already have knowns issues in a separate md file. I think we consolidate all the knowns issues per release into a single place.

Reply:
refactored existing known issue md file and placed issues into its own driver release pages.